tests/ttnn/unit_tests/operations/test_moreh_layer_norm.py

```diff
@@ -80,27 +81,28 @@ def tt_layer_norm(
 
     # dtype
     cpu_dtype = torch.bfloat16
+    npu_dtype = dtype
 
     # input
-    npu_input = to_ttnn(input, device=device)
+    npu_input = to_ttnn(input, device=device, dtype=npu_dtype)
```

Comment:
I think you should use `from_torch`

Reply:
I'll approve for now, please fix and I'll review again

Reply:
Me too.

Reply:
Since we're using `tensor([nan], dtype=torch.bfloat16)`, `from_torch` cannot create a tensor from this type of input, so I need to keep `to_ttnn`